### DISCUSSION TITLE
How to handle archives

### DISCUSSION THREAD

**Discussion by PaulieScanlon** [gatsby-uc/plugins#23] [General]
Would it be worth also advising folks how to archive their old un-maintained repos before moving to here?

I’m also wondering if something could be added to package.json keywords which Gatsby might be able to use to filter out un-maintained projects from the list on https://www.gatsbyjs.com/plugins

**Comment by moonmeister** [comment]
https://github.com/gatsby-uc/plugins/discussions/4#discussion-3264103 this is my current process, which does involve archiving the old repo. So far the plan has been to keep using the same npm package name. 

As for other retired packages, I do agree cleaning up the Gatsby plugin library could be beneficial but I don't work for Gatsby or have much say. If we wanted to put together some kind of RFC for improving that I'm sure they'd be happy to discuss aleast.

**Reply by KyleAMathews** [reply]
FYI, if a plugin is [deprecated](https://docs.npmjs.com/cli/v7/commands/npm-deprecate/), then it's filtered out of the plugin library. This is the intended way to prune away unmaintained or superseded plugins.

**Reply by KyleAMathews** [reply]
We can also manually mark plugins to not be shown which we've done a few times. There's no mechanism to do that but we could figure out a process if that became a standard thing e.g. for plugins where the maintainer is AWOL.

**Reply by PaulieScanlon** [reply]
Thanks both, good to know one end is taken care of in the Gatsbyjs site. Perhaps there's some docs or something that can be added to the README here which will help users understand how / why they should archive their old projects. 

Also i wonder if some more awareness work could be done for end users. I regularly see folks installing unmaintained plugins and becoming frustrated because it either has issues or sometimes because it's not v3 compatible 

Gatsbyjs does point out which plugins are official or community but i still notice users not understanding the difference and there's little in the way to help users understand if a community plugin is well maintained or not.
